casper/src/main/scala/coop/rchain/casper/genesis/contracts/RevGenerator.scala

```diff
@@ -1,63 +1,58 @@
 package coop.rchain.casper.genesis.contracts
 
-import coop.rchain.models.NormalizerEnv.NormalizerEnv
-import coop.rchain.models.{NormalizerEnv, Par}
+import coop.rchain.models.NormalizerEnv
 import coop.rchain.rholang.build.CompiledRholangSource
-import coop.rchain.rholang.interpreter.ParBuilder
-import monix.eval.Coeval
-
-final case class RevGenerator(userVaults: Seq[Vault], supply: Long) extends CompiledRholangSource {
 
+final class RevGenerator private (supply: Long, code: String)
+    extends CompiledRholangSource(code, NormalizerEnv.Empty) {
```

Comment:
This and the changes related to this should be in a separate commit.

Reply:
Definitely :+1: